src/components/Dialogs/TaskProgressDialog/TaskProgressDialogComponent.tsx

```diff
@@ -47,8 +48,11 @@ export class TaskProgressDialogComponent extends React.Component<TaskProgressDia
                 {this.props.cancellable && (
                     <div className={Classes.DIALOG_FOOTER}>
                         <div className={Classes.DIALOG_FOOTER_ACTIONS}>
+                            <span>{this.props.isCancelling && `(cancellation in progress${".".repeat(Math.floor(performance.now() / 1000) % 4).concat("\xa0".repeat(3 - (Math.floor(performance.now() / 1000) % 4)))} )`}</span>
```

Comment:
this should be neatened up a lot, and moved out of the `<div>`. You're re-using `Math.floor(performance.now() / 1000) % 4`

Reply:
done.